libvirt/tests/src/passthrough/pci/libvirt_pci_passthrough.py

```
@@ -112,6 +112,16 @@ def guest_lifecycle():
                            "ENTER.YOUR.SERVER.IP")
     netmask = params.get("libvirt_pci_net_mask", "ENTER.YOUR.Mask")
 
+    def netmask_to_cidr(netmask):
+        """Convert netmask like 255.255.255.0 to CIDR notation like 24"""
+        try:
+            octets = netmask.split(".")
+            binary = "".join([bin(int(octet))[2:].zfill(8) for octet in octets])
+            return str(binary.count("1"))
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)